protocol/0028-GOVE-governance.md

```diff
@@ -249,6 +249,19 @@ The following are immutable and cannot be changed:
 - position decimal places
 - `settlementAsset`
 
+### 2.1 Change market flags
+
+Separately to a general market change proposal, proposers can create a proposal to add or remove lists of free-text string flags to a market. Although these proposals change only a single market, it is possible that these tags will be used to control external behaviour of systems interacting with the vega chain, therefore they do not allow for voting using ELS and instead must be voted through by the general pool of token holders as with any other proposal. The maximum length of a flag is specified by the network parameter `governance.proposal.market.maxFlagLength`. The flags on a market are controlled by a single proposal type, `UpdateFlags` with a structure akin to:
```

Comment:
I think we'd like the voting on these to be a separate category with lengths and participations so:
```
governance.proposal.updateMarketFlags.maxClose
720h
governance.proposal.updateMarketFlags.maxEnact
720h
governance.proposal.updateMarketFlags.minClose
168h
governance.proposal.updateMarketFlags.minEnact
168h
governance.proposal.updateMarketFlags.minProposerBalance
20,000.00
governance.proposal.updateMarketFlags.minVoterBalance
1.00
governance.proposal.updateMarketFlags.requiredMajority
66%
governance.proposal.updateMarketFlags.requiredParticipation
```